### DISCUSSION TITLE
Set streaming platforms

### DISCUSSION THREAD

**Discussion by obxjames** [seerr-team/seerr#377] [Ideas]
So I have a weird one, I have family members who pay for streaming platforms. I do remind them they can just download but up to them. I'd like to add what platforms we have (Disney+, Prime video etc) so the request button will change to watch now. 

Think of it more as we use discover and see it's Disney+ the button can just take us direct to the movie rather than downloading it? Is this something that isn't just me that would find this useful?

**Comment by fallenbagel** [comment]
Do you mean redirect to that page in disney+ or sth for example? I'm not sure how we can even achieve that since there's not a database we can like scrape from for linking

**Reply by obxjames** [reply]
No need for direct to the video link, can go platform first as we already have the information that it's on Disney+ so button takes you to Disney+, maybe we can find a way in time to improve it, I've been looking at how and I have a few ideas. But no point complicating it if less than 1% of users would use it.
